india_compliance/gst_india/utils/e_waybill.py

```diff
@@ -469,7 +469,7 @@ def get_data(self):
         self.get_item_list()
         self.get_party_address_details()
 
-        return self.get_transaction_data()
+        return self.sanitize_data(self.get_transaction_data())
```

Comment:
this is already being done conditionally inside `get_transaction_data`. Why are we doing it here?